### PR TITLE
[Java] Extract out a new helper method from okhttp clients

### PR DESCRIPTION
**Pull request by brendandburns**
### Description of the PR
Extract out a method that will give you a `Request` object from existing `buildCall` call, useful for generating WebSockets requests.

### CODE REVIEW

File: modules/swagger-codegen/src/main/resources/Java/libraries/okhttp-gson/ApiClient.mustache [hunk 1]
```diff
@@ -1085,6 +1085,12 @@ public class ApiClient {
      * @throws ApiException If fail to serialize the request body object
      */
     public Call buildCall(String path, String method, List<Pair> queryParams, Object body, Map<String, String> headerParams, Map<String, Object> formParams, String[] authNames, ProgressRequestBody.ProgressRequestListener progressRequestListener) throws ApiException {
+        Request request = buildRequest(path, method, queryParams, body, headerParams, formParams, authNames, progressRequestListener);
+
+        return httpClient.newCall(request);
+    }
+
+    public Request buildRequest(String path, String method, List<Pair> queryParams, Object body, Map<String, String> headerParams, Map<String, Object> formParams, String[] authNames, ProgressRequestBody.ProgressRequestListener progressRequestListener) throws ApiException {
```
Comment (wing328) [inline modules/swagger-codegen/src/main/resources/Java/libraries/okhttp-gson/ApiClient.mustache]: @brendandburns thanks for the enhancement. I wonder if you can also add a docstring to `buildRequest`.

Comment (brendandburns) [inline modules/swagger-codegen/src/main/resources/Java/libraries/okhttp-gson/ApiClient.mustache]: done.

**Comment by brendandburns** [comment]
Thanks for the quick review, comment addressed.

**Comment by wing328** [comment]
@brendandburns thanks for the quick update. I'll merge it into master after all CI tests pass.

**Comment by wing328** [comment]
@brendandburns PR merged into master. Thanks for the enhancement.

**Pull request merged**